torchvision/models/efficientnet.py

```diff
@@ -32,15 +32,23 @@
 
 
 class SqueezeExcitation(nn.Module):
-    def __init__(self, input_channels: int, squeeze_channels: int):
+    def __init__(
+        self,
+        input_channels: int,
+        squeeze_channels: int,
+        activation: Optional[Callable[..., nn.Module]] = None,
+    ) -> None:
         super().__init__()
         self.fc1 = nn.Conv2d(input_channels, squeeze_channels, 1)
         self.fc2 = nn.Conv2d(squeeze_channels, input_channels, 1)
+        if activation is None:
+            activation = nn.SiLU
+        self.activation = activation(inplace=True)
 
     def _scale(self, input: Tensor) -> Tensor:
         scale = F.adaptive_avg_pool2d(input, 1)
         scale = self.fc1(scale)
-        scale = F.silu(scale, inplace=True)
+        scale = self.activation(scale)
         scale = self.fc2(scale)
         return scale.sigmoid()
```

Comment:
[MobileNetV3](https://github.com/pytorch/vision/blob/main/torchvision/models/mobilenetv3.py#L35) uses `hardsigmoid` here, so we probably need another option to allow someone replacing sigmoid with another activation.